server/eclipse-project/src/main/java/us/freeandfair/corla/json/CountyDashboardRefreshResponse.java

```diff
@@ -152,30 +160,34 @@ protected CountyDashboardRefreshResponse(final Integer the_id,
                                            final CountyStatus the_status,
                                            final Map<String, String> the_general_information,
                                            final Set<Elector> the_audit_board_members, 
-                                           final String the_ballot_manifest_digest,
-                                           final String the_cvr_export_digest,
+                                           final String the_manifest_digest,
+                                           final Instant the_manifest_timestamp,
+                                           final String the_cvr_digest,
+                                           final Instant the_cvr_timestamp,
                                            final Set<Long> the_contests,
                                            final Map<Long, String> the_contests_under_audit,
                                            final Instant the_audit_time,
                                            final Integer the_estimated_ballots_to_audit,
-                                           final Integer the_number_of_ballots_audited,
-                                           final Integer the_number_of_discrepencies, 
-                                           final Integer the_number_of_disagreements,
+                                           final Integer the_ballots_audited,
+                                           final Integer the_discrepencies, 
```

Comment:
This is a typo (`s/the_discrepencies/the_discrepancies`), but should actually become `the_discrepancy_count`, given my above comments.